conbench/templates/app.html

```diff
@@ -86,8 +86,8 @@ <h4 class="modal-title"><i class="bi bi-trash3-fill"></i></h4>
         {% if messages %}
             <!-- Categories: success (green), info (blue), warning (yellow), danger (red) -->
             {% for category, message in messages %}
-            <div class="alert alert-{{ category }}" role="alert">
-                <p class="fs-5"><i class="bi bi-exclamation-diamond-fill"></i> {{ message }}</p>
+            <div class="fs-5 alert alert-{{ category }} text-center" role="alert">
+                <i class="bi bi-info-circle"></i> {{ message }}
```

Comment:
Sneaked a tiny UI fix in here. 

The icon is better, the fs-5 on the p wasn't good (resulting in asymmetric padding)